sdk/search/azure-search-documents/azure/search/documents/indexes/_internal/_search_index_client.py

```diff
@@ -380,11 +380,11 @@ def create_synonym_map(self, name, synonyms, **kwargs):
         solr_format_synonyms = "\n".join(synonyms)
         synonym_map = SynonymMap(name=name, synonyms=solr_format_synonyms)
         result = self._client.synonym_maps.create(synonym_map, **kwargs)
-        return listize_synonyms(result.as_dict())
+        return listize_synonyms(result)
```

Comment:
Here (and everywhere else that calls `listized_synonyms`) this will cause the returned result not to match the documentation on `SynonymMap`:
```
    :param synonyms: Required. A series of synonym rules in the specified synonym map format. The
     rules must be separated by newlines.
    :type synonyms: str
```

Reply:
Well, I guess *technically* that is the doc for the parameter to the initializer and is therefore still *technically* correct. But as a user wanting to know what the value of `some_map.synonyms` should be, I think I would be misled

Reply:
This is a good point. I added the wrapper. Fixed.